gaurav/src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import React from "react";
 import { MarkTime } from "./components/MarkTime";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import { Root } from "./routes/Root";
+import { Log } from "./components/Log";
 function App() {
-  return <div className="font-sans flex justify-center items-center">
-    <MarkTime />
-  </div>;
+  const router = createBrowserRouter([
+    {
+      path: "/",
+      element: <Root />,
+      children: [
+        {
+          path: "timelog",
+          element: <MarkTime />,
+        },
+        {
```

Comment:
no point of creating nested routes if we have just two component files

Reply:
Removed nested Routes.